test/TestUtilities/Test.Utility/Signing/CertificateAuthority.cs

```diff
@@ -195,30 +198,53 @@ internal CertStatus GetCertStatus(CertId certId, out X509Certificate2 certificat
         {
             certificate = null;
 
-            if (certId.AlgorithmIdentifier.Algorithm.Value != Oids.Sha1)
+            Oid hashAlgorithmOid = certId.AlgorithmIdentifier.Algorithm;
+
+            if (hashAlgorithmOid.Value == Oids.Sha1)
             {
-                return CertStatus.FromUnknown();
+                // SHA-1 is obsolete.  We'll rely on serial number only for matching.
```

Comment:
Should we return `CertStatus.FromUnknown();` in this case?

Reply:
Nope.  As I mentioned in the description of this PR:

> We can ban SHA-1 in our test TSA; however, we can't ban SHA-1 in OCSP clients. We don't control that. We have tests that chain build and check revocation status and it's the operating system --- in this case, Windows --- that chooses SHA-1 to identify a certificate in the OCSP request.

If we returned `CertStatus.FromUnknown()` here, OCSP revocation checking would generally fail on Windows.